ui/src/data/source.tsx

```diff
@@ -1713,8 +1723,21 @@ function processAttributes(
 ) {
   for (const k in attributes) {
     const v = attributes[k];
-    const value = dataValueFromLiteral(v.value);
-    obj[k] = v.display ?? value;
+    let value: DataValue = null;
+    let display = v.display;
+
+    if (v.isRepeatedValue) {
+      // TODO(tjennison): Consider how best to support repeated values beyond
```

Comment:
nit: consider jira ticket ids for TODOs

Reply:
I feel like that's better for work we actually intend to do. This is more just a comment for anyone who wonders why these values are hardcoded in the future. Maybe a plain comment would have been the right choice..